ibcdfo_pypkg/tox.ini

```diff
@@ -0,0 +1,64 @@
+#
+# Setup and manage python virtual environments that allow users to develop and
+# test the ibcdfo python package.
+#
+
+[tox]
+requires = tox>=4
+env_list = nocoverage
+
+[testenv]
+allowlist_externals = wget
+description = Run the ibcdfo full test suite
+basepython = py3
+passenv = COVERAGE_FILE, PYTHONPATH
+setenv =
+    COV_FILE = {env:COVERAGE_FILE:.coverage_ibcdfo}
+deps =
+    coverage: coverage
+usedevelop =
+    coverage: true
+    nocoverage: false
+commands =
+    wget https://raw.githubusercontent.com/POptUS/BenDFO/main/data/dfo.dat -O dfo.dat
+    nocoverage: python -m unittest ibcdfo -v
+    coverage: coverage run --data-file={env:COV_FILE} -m unittest discover -s ./src/ibcdfo -t .. -v
+
+[testenv:pounders]
+description = Run the ibcdfo pounders test suite
+commands = python -m unittest ibcdfo.pounders
+
+# [testenv:subB]
+# description = Run the ibcdfo manifold_sampling test suite
+# commands = python -m unittest ibcdfo.manifold_sampling
+
+[testenv:report]
+description = Generate coverage reports
+depends = coverage
+deps = coverage
+skip_install = true
+commands =
+    coverage report --data-file={env:COV_FILE}
+    coverage html --data-file={env:COV_FILE}
+
+[testenv:testdeploy]
+description = Deploy distribution to TestPyPi for testing
+deps = twine
+commands =
+    python setup.py sdist
+    python setup.py bdist_wheel
+    # This will prompt for username/password
+    #
+    # If dist was not empty beforehand, this could potentially upload many
+    # different versions of distributions.
+    twine upload --repository testpypi dist/*
```

Comment:
FYI, I never tested this section.  Note sure if you need it.

Reply:
I have removed it for now.